Configurations/50-win-onecore.conf

```diff
@@ -0,0 +1,20 @@
+# Configurations using Windows OneCore API
+# OneCore build should link only via one of the OneCore libs:
+# OneCore.lib, OneCoreUAP.lib, OneCore_downlevel.lib, OneCoreUAP_downlevel.lib
```

Comment:
I find it a bit misleading. It says that it's sufficient to link with *one* lib [of developer's choice], but I failed to link with onecore_downlevel.lib alone [on ARM], I had to specify onecore.lib onecore_downlevel.lib. Can *you* actually link with onecore_downlevel.lib alone?

Reply:
Oh i meant that generally onecore is about linking via these lib's it does not meant it will work in this particular case (openssl) cause it is specified by used api, in openssl case only onecore.lib is wide enough to work.
I will change that comment it is misleading